_data/simple-icons.json

```
@@ -419,6 +419,17 @@
             "hex": "04ACE6",
             "source": "https://www.ah.nl"
         },
+        {
+            "title": "Alby",
+            "hex": "FFDF6F",
+            "source": "https://github.com/getAlby/media/blob/28fd7aa05fee15777dfdf2af2166d05b8f5f687f/Alby-logo-icons/Alby-logo-head/alby.svg",
+            "guidelines": "https://github.com/getAlby/lightning-browser-extension/wiki/Open-Design",
+            "aliases": {
+                "aka": [
+                    "getAlby"
+                ]
+            }
```

```suggestion
            "guidelines": "https://github.com/getAlby/lightning-browser-extension/wiki/Open-Design"
```

I'd argue the alias isn't required, as the full title is visible as part of it. Highly unlikely that someone searching for this will just start with 'get' and expect a result! What do other @simple-icons/maintainers think?